.github/workflows/test.yaml

```diff
@@ -0,0 +1,49 @@
+name: Tests
+on:
+  push:
+
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - "3.8"
```

Comment:
Added this here just because I'm planning to work on python 3.9 support next. I can also not have the matrix if preferred.

Reply:
for reference, this is the current issue with 3.9:

```
self = <pynamodb_attributes.unicode_delimited_tuple.UnicodeDelimitedTupleAttribute object at 0x10ddb2cd0>, value = 'US.San Francisco.94107'
    def deserialize(self, value: str) -> T:
        fields = getattr(self.tuple_type, '_fields', None)
        field_types = getattr(self.tuple_type, '_field_types', None)
        if fields and field_types:
            values = value.split(self.delimiter, maxsplit=len(fields))
            return self.tuple_type(**{f: field_types[f](v) for f, v in zip(fields, values)})
        else:
>           return self.tuple_type(value.split(self.delimiter))
E           TypeError: <lambda>() missing 1 required positional argument: 'city'
pynamodb_attributes/unicode_delimited_tuple.py:48: TypeError
```

_field_types is gone: https://bugs.python.org/issue40182